content/release-notes/2023-05-09-storage.md

```diff
@@ -0,0 +1,12 @@
+---
+label: 'Storage'
+---
+
+### What's new
+
+- Compute: Docker image files are now included in the [neon](https://github.com/neondatabase/neon) GitHub repository.
+- Safekeepers: Enabled parallel offload of WAL segments to remote storage. This change allows Safekeepers to upload up to five segments concurrently.
+
+### Bug fixes
+
+- Compute: Addressed an issue that prevented PostgreSQL 15 from entering standby mode.
```

Comment:
It's related to replication. However, I'm not sure that it's the right explanation.

@lubennikovaav, @MMeent, can you help?

Reply:
Thank you. Let's remove it until we release the feature.